Felienne/hedy
# [FIX] Differentiates between search as placeholder and as button string

**Description**
In this PR we add an extra translation to make sure search buttons are no longer labelled as "search..." but simply as "search".

**Fixes**
This PR fixes #2318

**How to test**
Navigate to either the explore or programs page and verify that the search button now indeed only reads "search".
